### ISSUE TITLE
PRELOAD_MODULES no longer works

### ISSUE THREAD

**Issue by raboof** [raboof/notion#203]
the fontconfig library cannot be found when building like this.

**Comment by wilhelmy** [comment]
Do we need fontconfig outside of Xft builds?

**Comment by raboof** [comment]
No, this problem is specific to builds that have XFT enabled

**Comment by raboof** [comment]
(though it's enabled by default, so that's probably common ;) )

**Comment by wilhelmy** [comment]
Ah, I see. The distribution package in /usr/bin still doesn't have it enabled (2017050501-1)  and neither does my own build (although I don't remember explicitly disabling it during build).

**Comment by raboof** [comment]
> I didn't realize only de.so is linked against Xft, not the notion binary itself.

Yeah that threw me off while debugging this one as well ;). And is likely related to the reason it's not working in the PRELOAD_MODULES case now.

**Comment by wilhelmy** [comment]
Is PRELOAD_MODULES enabled by default? Should we just remove or disable it?

**Comment by raboof** [comment]
It is not enabled by default, making this a lower-priority issue. I seem to remember it was required on one of the lesser-used targets (perhaps solaris?), so perhaps good to keep it in so it can be fixed when someone wants to look into this.
